docs/source/object_detection.mdx

```diff
@@ -154,6 +154,9 @@ You can verify the transform works by visualizing the 10th example:
 ... )
 ```
 
+To learn how to leverage parts of 🤗 datasets end-to-end for training an object detection model, refer to
```

Comment:
This should be after the image in the following line

Reply:
Done.